displaytag/pom.xml

```diff
@@ -20,7 +20,7 @@
   </scm>
 
   <properties>
-    <tomcat.version>7.0.109</tomcat.version>
+    <tomcat.version>11.0.0-M1</tomcat.version>
```

Comment:
@allantodd Make this 10.1.x series not 11 milestone.

Reply:
done